src/Symfony/Component/HttpFoundation/Request.php

```diff
@@ -250,8 +258,15 @@ class Request
      * @param array                $server     The SERVER parameters
      * @param string|resource|null $content    The raw body data
      */
-    public function __construct(array $query = [], array $request = [], array $attributes = [], array $cookies = [], array $files = [], array $server = [], $content = null)
-    {
+    public function __construct(
```

Comment:
all such changes must be reverted: our CS is preferring long lines over split ones.